_posts/2021-04-19-security-incident-disclosure.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Security Incident Disclosure
+author: Markus Reiter
+---
+
+On 18th April 2021 a security researcher identified a vulnerability in our `review-cask-pr` GitHub action which is used to automatically review and approve version bump pull requests. This was reported via Hacker One and [a pull request demonstrating the vulnerability](https://github.com/Homebrew/homebrew-cask/pull/104191) was submitted in accordance with us. The [pull request was subsequently reverted](https://github.com/Homebrew/homebrew-cask/pull/104197) and the [`review-cask-pr` action disabled and removed](https://github.com/Homebrew/homebrew-cask/pull/104204) from all repositories that were using it.
```

Comment:
Can we additionally note:
- it was merged by the automerge action and that it has also been disabled in favour of the GitHub feature in all homebrew-cask* repositories
- we've removed the ability for our bot(s) to commit to homebrew-cask* repositories
- we are moving forward with a more manual review process
- we are upskilling homebrew-core maintainers and documenting the process to make it easier to onboard homebrew-cask maintainers